router.js

```diff
@@ -45,9 +45,9 @@ router.get("/load", async (ctx) => {
   d = d.replace("dataPlaneUrl", dataPlaneUrl);
   d = d.replace("configBackendUrl", configUrl);
 
-  deviceModeInit = deviceModeInit.replaceAll("dataplaneUrl", dataPlaneUrl);
-  deviceModeInit = deviceModeInit.replaceAll("writeKey_placeHolder", writeKey);
-  deviceModeInit = deviceModeInit.replaceAll("configUrl_placeholder", configUrl);
+  deviceModeInit = deviceModeInit.replace(/dataplaneUrl/g, dataPlaneUrl);
```

Comment:
Do we have test-case for this ? I mean not literally for `replace(/dataplaneUrl/g, "")` but more like expressing the intention of this kind of replace implementation.

Reply:
Didn't understand your point. Will changing it to `dataplaneUrl_placeHolder` would make sense?